[2024-06-03T17:39:12.588+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T17:39:12.605+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-03T17:39:12.610+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-03T17:39:12.611+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T17:39:12.621+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2021-04-01 00:00:00+00:00
[2024-06-03T17:39:12.626+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=203) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T17:39:12.627+0000] {standard_task_runner.py:63} INFO - Started process 207 to run task
[2024-06-03T17:39:12.627+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2021-04-01T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpurpzl58x']
[2024-06-03T17:39:12.628+0000] {standard_task_runner.py:91} INFO - Job 48: Subtask games_etl
[2024-06-03T17:39:12.663+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [running]> on host a5a4421409ea
[2024-06-03T17:39:12.720+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-01T00:00:00+00:00'
[2024-06-03T17:39:12.721+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T17:39:12.722+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T17:39:13.141+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T17:39:13.146+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/chess_dag.py", line 19, in games_etl_task
    return get_monthly_games(execution_date)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/chess_etl_code.py", line 60, in get_monthly_games
    connection = BaseHook.get_connection('chess_project_s3')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `chess_project_s3` isn't defined
[2024-06-03T17:39:13.151+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2021-04-01T00:00:00+00:00, execution_date=20210401T000000, start_date=20240603T173912, end_date=20240603T173913
[2024-06-03T17:39:13.158+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 48 for task games_etl (The conn_id `chess_project_s3` isn't defined; 207)
[2024-06-03T17:39:13.202+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-03T17:39:13.216+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T17:39:13.219+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-03T21:59:07.356+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T21:59:07.373+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-03T21:59:07.379+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-03T21:59:07.380+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T21:59:07.390+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2021-04-01 00:00:00+00:00
[2024-06-03T21:59:07.397+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=27502) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T21:59:07.398+0000] {standard_task_runner.py:63} INFO - Started process 27504 to run task
[2024-06-03T21:59:07.399+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2021-04-01T00:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpe40dbhfv']
[2024-06-03T21:59:07.400+0000] {standard_task_runner.py:91} INFO - Job 118: Subtask games_etl
[2024-06-03T21:59:07.437+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T21:59:07.505+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-01T00:00:00+00:00'
[2024-06-03T21:59:07.506+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T21:59:07.507+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T21:59:07.871+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T21:59:08.455+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T21:59:08.455+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T21:59:08.464+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2021-04-01T00:00:00+00:00, execution_date=20210401T000000, start_date=20240603T215907, end_date=20240603T215908
[2024-06-03T21:59:08.494+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T21:59:08.509+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-03T21:59:08.511+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T19:26:09.438+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T19:26:09.454+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-08T19:26:09.459+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-06-08T19:26:09.459+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T19:26:09.469+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2021-04-01 00:00:00+00:00
[2024-06-08T19:26:09.476+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=27910) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T19:26:09.478+0000] {standard_task_runner.py:63} INFO - Started process 27912 to run task
[2024-06-08T19:26:09.478+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2021-04-01T00:00:00+00:00', '--job-id', '294', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpdodutha4']
[2024-06-08T19:26:09.479+0000] {standard_task_runner.py:91} INFO - Job 294: Subtask games_etl
[2024-06-08T19:26:09.515+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T19:26:09.584+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-01T00:00:00+00:00'
[2024-06-08T19:26:09.585+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T19:26:09.586+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T19:26:10.029+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T19:26:10.883+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T19:26:10.884+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T19:26:10.891+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2021-04-01T00:00:00+00:00, execution_date=20210401T000000, start_date=20240608T192609, end_date=20240608T192610
[2024-06-08T19:26:10.935+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T19:26:10.951+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-08T19:26:10.952+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T01:24:02.674+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T01:24:02.689+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-07-08T01:24:02.695+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [queued]>
[2024-07-08T01:24:02.696+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T01:24:02.706+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): games_etl> on 2021-04-01 00:00:00+00:00
[2024-07-08T01:24:02.713+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=28728) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T01:24:02.715+0000] {standard_task_runner.py:63} INFO - Started process 28738 to run task
[2024-07-08T01:24:02.715+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'games_etl', 'scheduled__2021-04-01T00:00:00+00:00', '--job-id', '664', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp24h52z42']
[2024-07-08T01:24:02.716+0000] {standard_task_runner.py:91} INFO - Job 664: Subtask games_etl
[2024-07-08T01:24:02.753+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.games_etl scheduled__2021-04-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T01:24:02.827+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='games_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-01T00:00:00+00:00'
[2024-07-08T01:24:02.828+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T01:24:02.829+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T01:24:03.217+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T01:24:03.781+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T01:24:03.783+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T01:24:03.790+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=games_etl, run_id=scheduled__2021-04-01T00:00:00+00:00, execution_date=20210401T000000, start_date=20240708T012402, end_date=20240708T012403
[2024-07-08T01:24:03.811+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T01:24:03.830+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T01:24:03.831+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
